samples/sample13/README.md

```diff
@@ -11,143 +11,240 @@
+            │   │   |   |   └── hot_drink.bir                      
+            └── jar_cache                                      
+                └── kathy                                       
+            │   │   ├── cool_drink                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-cool_drink-0.0.1.jar
+            │   │   ├── drink_store                                 
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-drink_store-0.0.1.jar                     
+            │   │   └── hot_drink                                  
+            │   │   |   └── 0.0.1                              
+            │   │   |   |   └── kathy-hot_drink-0.0.1.jar
   
     ```
 ### How to run:
-1. Go to `resource/docker` folder and run the build.sh file. This will generate two databases deployment in mysql namespace.
+1. Use the link [https://jar-download.com/artifacts/mysql/mysql-connector-java/8.0.11/source-code] to download [mysql-connector-java-8.0.11.jar] and add it to a new folder named `libs` inside sample13 directory.
+2. Go to `resource/docker` folder and run the build.sh file. This will generate two databases deployment in mysql namespace.
 Verify pods and services are created. 
 
 ```bash
 $> ./build.sh
 Sending build context to Docker daemon  3.584kB
 Step 1/2 : FROM mysql:5.7.22
- ---> 66bc0f66b7af
+ ---> 6bb891430fb6
 Step 2/2 : COPY script.sql /docker-entrypoint-initdb.d/
  ---> Using cache
- ---> 08789a90af68
-Successfully built 08789a90af68
+ ---> 6114e58c6b53
+Successfully built 6114e58c6b53
 Successfully tagged hotdrink_mysql_db:1.0.0
 Sending build context to Docker daemon  3.584kB
 Step 1/2 : FROM mysql:5.7.22
- ---> 66bc0f66b7af
+ ---> 6bb891430fb6
 Step 2/2 : COPY script.sql /docker-entrypoint-initdb.d/
  ---> Using cache
- ---> ca5db463028b
-Successfully built ca5db463028b
+ ---> f67a133230f3
+Successfully built f67a133230f3
 Successfully tagged cooldrink_mysql_db:1.0.0
-namespace "mysql" created
-deployment.extensions "hotdrink-mysql-deployment" created
-service "hotdrink-mysql" created
-deployment.extensions "cooldrink-mysql-deployment" created
-service "cooldrink-mysql" created
+namespace/mysql created
+deployment.apps/hotdrink-mysql-deployment created
+service/hotdrink-mysql created
+deployment.apps/cooldrink-mysql-deployment created
+service/cooldrink-mysql created
 
 $> kubectl get pods -n mysql
 NAME                                          READY     STATUS    RESTARTS   AGE
 cooldrink-mysql-deployment-587c466765-zs9ll   1/1       Running   0          39s
 hotdrink-mysql-deployment-77869db489-c8lcf    1/1       Running   0          39s
 ```
 
-1. Navigate to sample13 folder and Initialize ballerina project.
+1. Navigate to sample13 folder and compile the project. Commands to run kubernetes artifacts will be printed on success:
```

Comment:
It should be "build the project". 
Also this: "Commands to deploy kubernetes artifacts will be printed on build success."

Reply:
Updated.